packages/quantic/force-app/main/default/lwc/quanticCitation/__tests__/quanticCitation.test.js

```diff
@@ -260,6 +262,9 @@ describe('c-quantic-citation', () => {
       await citationLink.dispatchEvent(
         new CustomEvent('mouseenter', {bubbles: true})
       );
+      // Wait for show delay - this triggers the tooltip to show and sets hoverStartTimestamp
+      jest.advanceTimersByTime(200);
+      // Now wait for minimum hover time
       jest.advanceTimersByTime(1000);
       await citationLink.dispatchEvent(
         new CustomEvent('mouseleave', {bubbles: true})
```

Comment:
in line 272, we have:
```
// Additional 200ms delay for the new hide tooltip logic
jest.advanceTimersByTime(200);
```
That's no longer not accurate now, because closeTooltipDelayMs is 100.